omnibus/config/software/rb-fsevent-gem.rb

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2012-2014 Chef Software, Inc.
```

Comment:
This is going to seem really silly, but can we get this to be 2020 not 2012-2014